pkg/arvo/neo/cod/std/src/fil/developer-environment-setup.txt

```diff
@@ -0,0 +1,16 @@
+# Developer Environment Setup
+
+The quickest way to get set up to play with Shrubbery and Sky is to boot a comet and run `|install ~met %neo`.
```

Comment:
Link to urbit.org guide for booting a comet

Include instructions for booting a moon

Mention `--loom 33` and explain why it's used